README.md

```diff
@@ -8,7 +8,15 @@ Performs a `pg_dump` against a source database and restores it into the specifie
 
 1. Clone this project and update the Dockerfile's `PG_VERSION` to match the version of your target Postgres cluster.
 
-2. Specify the `SOURCE_DATABASE_URI` and `TARGET_DATABASE_URI` as secrets.
+2. Run `fly apps create --name postgres-migration-tool`
+
+3. Create a volume called `migration_data` in the same region as you did when you ran `fly apps create`
+
+```
+fly volumes create migration_data --region iad --size 20 -a postgres-migration-tool
```

Comment:
Were you not able to run your migration without a Volume?  Given how we are piping `pg_dump` to `psql`  a Volume shouldn't actually be necessary.

https://github.com/fly-apps/postgres-migrator/blob/main/migrate#L4

Reply:
Hey! I'll try without it :)